<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowFunction</class>
 <widget class="QDialog" name="WindowFunction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>477</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Function</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: rgba(200, 200, 200, 0.95);
    border: none;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButtonPaint">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background: #00b4d8; color: #222222; border: 1px solid #0096c7; border-bottom: 3px solid #0096c7; border-radius: 8px; padding: 10px 20px; font: bold 14px; } QPushButton:hover { background: #48cae4; border-bottom: 3px solid #00a8e8; } QPushButton:pressed { background: #0096c7; border-bottom: 1px solid #0077b6; margin-top: 2px; }</string>
      </property>
      <property name="text">
       <string>Закраска</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButtonSearchWidth">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background: #00b4d8; color: #222222; border: 1px solid #0096c7; border-bottom: 3px solid #0096c7; border-radius: 8px; padding: 10px 20px; font: bold 14px; } QPushButton:hover { background: #48cae4; border-bottom: 3px solid #00a8e8; } QPushButton:pressed { background: #0096c7; border-bottom: 1px solid #0077b6; margin-top: 2px; }</string>
      </property>
      <property name="text">
       <string>Поиск в
ширину</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButtonAdjVertex">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background: #00b4d8; color: #222222; border: 1px solid #0096c7; border-bottom: 3px solid #0096c7; border-radius: 8px; padding: 10px 20px; font: bold 14px; } QPushButton:hover { background: #48cae4; border-bottom: 3px solid #00a8e8; } QPushButton:pressed { background: #0096c7; border-bottom: 1px solid #0077b6; margin-top: 2px; }</string>
      </property>
      <property name="text">
       <string>Матрица
смежности
вершин</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButtonAdjEdge">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background: #00b4d8; color: #222222; border: 1px solid #0096c7; border-bottom: 3px solid #0096c7; border-radius: 8px; padding: 10px 20px; font: bold 14px; } QPushButton:hover { background: #48cae4; border-bottom: 3px solid #00a8e8; } QPushButton:pressed { background: #0096c7; border-bottom: 1px solid #0077b6; margin-top: 2px; }</string>
      </property>
      <property name="text">
       <string>Матрица
смежности
рёбер</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButtonDeikstra">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background: #00b4d8; color: #222222; border: 1px solid #0096c7; border-bottom: 3px solid #0096c7; border-radius: 8px; padding: 10px 20px; font: bold 14px; } QPushButton:hover { background: #48cae4; border-bottom: 3px solid #00a8e8; } QPushButton:pressed { background: #0096c7; border-bottom: 1px solid #0077b6; margin-top: 2px; }</string>
      </property>
      <property name="text">
       <string>Алгоритм
Дейкстры</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButtonIncidence">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background: #00b4d8; color: #222222; border: 1px solid #0096c7; border-bottom: 3px solid #0096c7; border-radius: 8px; padding: 10px 20px; font: bold 14px; } QPushButton:hover { background: #48cae4; border-bottom: 3px solid #00a8e8; } QPushButton:pressed { background: #0096c7; border-bottom: 1px solid #0077b6; margin-top: 2px; }</string>
      </property>
      <property name="text">
       <string>Матрица
инциден-
тности</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButtonFord">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background: #00b4d8; color: #222222; border: 1px solid #0096c7; border-bottom: 3px solid #0096c7; border-radius: 8px; padding: 10px 20px; font: bold 14px; } QPushButton:hover { background: #48cae4; border-bottom: 3px solid #00a8e8; } QPushButton:pressed { background: #0096c7; border-bottom: 1px solid #0077b6; margin-top: 2px; }</string>
      </property>
      <property name="text">
       <string>Алгоритм
Форда</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButtonExit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #ff5555;
    color: #000000;          
    border: 2px solid #cc0000;
    border-radius: 6px;
    padding: 8px 16px;
    font: bold 14px 'Arial';
    min-width: 100px;
    outline: none;
}
QPushButton:hover {
    background-color: #ff7777;
    border-color: #dd2222;
}

QPushButton:pressed {
    background-color: #dd2222;
    border-color: #bb1111;
    color: #111111;
}

QPushButton:disabled {
    background-color: #aa9999;
    color: #555555;
    border-color: #998888;
}</string>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButtonSearchDepth">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background: #00b4d8; color: #222222; border: 1px solid #0096c7; border-bottom: 3px solid #0096c7; border-radius: 8px; padding: 10px 20px; font: bold 14px; } QPushButton:hover { background: #48cae4; border-bottom: 3px solid #00a8e8; } QPushButton:pressed { background: #0096c7; border-bottom: 1px solid #0077b6; margin-top: 2px; }</string>
      </property>
      <property name="text">
       <string>Поиск в
глубину</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButtonCodeP">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background: #00b4d8; color: #222222; border: 1px solid #0096c7; border-bottom: 3px solid #0096c7; border-radius: 8px; padding: 10px 20px; font: bold 14px; } QPushButton:hover { background: #48cae4; border-bottom: 3px solid #00a8e8; } QPushButton:pressed { background: #0096c7; border-bottom: 1px solid #0077b6; margin-top: 2px; }</string>
      </property>
      <property name="text">
       <string>Код
Прюфера</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
